review-tools/README

```diff
@@ -115,6 +115,9 @@ else 'git merge --no-commit --squash'.
 It works on the current branch, which should be master or one of the stable
 releases. For the branch to be merged a suitable remote must be available.
 
+The tool accesses databases on api.github.com and www.openssl.org via HTTPS.
```

Comment:
www.openssl.org is incorrect.

Reply:
So should I just remove "and www.openssl.org", 
or which other URLs are (potentially) contacted besides api.github.org?

Reply:
Why not just write

    The tool accesses external databases via HTTPS (e.g., api.github.com).

since your main point is to inform the reader about the proxy settings?

Reply:
Done